catalyst/rl/wrappers/raw_trajectory.py

```diff
@@ -0,0 +1,122 @@
+from gym.core import Wrapper
+from catalyst.rl.utils.buffer import get_buffer
+from catalyst.utils.dynamic_array import DynamicArray
+import numpy as np
+
+
+class RawTrajectoryWrapper(Wrapper):
+    def __init__(
+            self,
+            env,
+            allow_early_resets=False,
+            initial_capacity=1e3,
+            reset_keywords=(),
+            info_keywords=()
```

Comment:
why do we need `reset_keywords` and `info_keywords`?

Reply:
`reset_keywords` are needed if we want to pass some arguments to the reset function, probably something like `env.reset(difficulty=1)`, and `info_keywords` seems not to be required.

Reply:
why should this wrapper check the reset arguments?
I think, this feature should be somewhere in the EnvWrapper [here](https://github.com/catalyst-team/catalyst/blob/master/catalyst/rl/environment/gym.py#L55), for example